perceval/backend.py

```diff
@@ -510,6 +539,157 @@ def setup_cmd_parser(cls):
         raise NotImplementedError
 
 
+class Summary:
+
+    def __init__(self):
+        self._items_fetched = 0
+        self._items_skipped = 0
+        self._min_updated_on = None
+        self._max_updated_on = None
+        self._last_updated_on = None
+        self._last_uuid = None
+        self._min_offset = None
+        self._max_offset = None
+        self._extras = None
+
+    @property
```

Comment:
I think we can remove all this setters and getters. Since the main method to update the summary is `update` and it encapsulates all the logic, I don't think we need all this code.

What it would be useful is to have a property like `total` which returns the sum of `fetched` and `skipped`.